querybook/server/models/query_execution.py

```diff
@@ -188,6 +188,29 @@ def to_dict(self):
         }
 
 
+class QueryExecutionMetadata(CRUDMixin, Base):
+    __tablename__ = "query_execution_metadata"
+
+    id = sql.Column(sql.Integer, primary_key=True)
+    query_execution_id = sql.Column(
+        sql.Integer, sql.ForeignKey("query_execution.id", ondelete="CASCADE")
+    )
+    execution_metadata = sql.Column(sql.JSON)
```

Comment:
can we add some docstring about what the metadata look like? currently it is only has the sample_rate

Reply:
added!